### PR TITLE
Misk to Wisp for Containers Testing code

### PR DESCRIPTION
**Pull request by chris-ryan-square**

### CODE REVIEW

File: misk-testing/src/main/kotlin/misk/containers/Containers.kt [hunk 1]
```diff
@@ -66,154 +61,19 @@ data class Container(
...
-  private class GracefulWaitContainerResultCallback : WaitContainerResultCallback() {
-    override fun onError(throwable: Throwable?) {
-      // this is ok, just meant that the container already terminated before we tried to wait
-      if (throwable is NotFoundException) {
-        return
-      }
-      super.onError(throwable)
-    }
-  }
+  var delegate: wisp.containers.Composer
```
Comment (r3mariano) [inline misk-testing/src/main/kotlin/misk/containers/Containers.kt:64]: nit: Something similar to this so `delegate` is a `val`? (Also, `private/internal val`?)
```kotlin
val delegate: Composer = run {
  // ... init code
}
```
?

File: wisp-containers-testing/src/main/kotlin/wisp/containers/Containers.kt [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+ *         .withExposedPorts(ExposedPort.tcp(port))
+ *         .withPortBindings(Ports().apply {
+ *           bind(ExposedPort.tcp(9102), Ports.Binding.bindPort(9102))
+ *         })
+ *         .withEnv(
+ *           "KAFKA_ZOOKEEPER_CONNECT=zookeeper:2181",
+ *           "KAFKA_ADVERTISED_LISTENERS=PLAINTEXT://localhost:9102")
+ *         }
+ *     val composer = Composer("e-kafka", zkContainer, kafka)
+ *     composer.start()
```
Comment (r3mariano) [inline wisp-containers-testing/src/main/kotlin/wisp/containers/Containers.kt:64]: Is this a new example? It's possible to do more syntax sugar with the existing code:
```kotlin
val zkContainer = Container {
    withImage("confluentinc/cp-zookeeper")
    withName("zookeeper")
    withEnv("ZOOKEEPER_CLIENT_PORT=2181")
 }
```
or an infix equivalent, but this seems fine.

**Review by r3mariano — APPROVED** [review]

File: misk-testing/src/main/kotlin/misk/containers/Containers.kt (hunk 1 above)
Comment (chris-ryan-square) [reply misk-testing/src/main/kotlin/misk/containers/Containers.kt:64]: Yeah, should have changed that - will fix.

File: wisp-containers-testing/src/main/kotlin/wisp/containers/Containers.kt (hunk 2 above)
Comment (chris-ryan-square) [reply wisp-containers-testing/src/main/kotlin/wisp/containers/Containers.kt:64]: Existing comments, just transferred, but that's better, will fix

**Pull request merged**